Remotion/Web/ClientScript/Scripts/ViewLayout.ts

```diff
@@ -16,47 +16,63 @@
 // 
 class ViewLayout
 {
-  public static AdjustActiveViewContent (viewContent: JQuery): void
+  public static AdjustActiveViewContent (viewContent: HTMLElement): void
   {
-    var viewContentBorder = viewContent.children('div').eq (0);
-    var viewBottomControls = viewContent.next();
+    var viewContentBorder = viewContent.querySelector ('div')!;
+    var viewBottomControls = viewContent.nextElementSibling as HTMLElement;
 
-    var viewContentBorderHeight = viewContentBorder.outerHeight (true) - viewContentBorder.height();
-    var viewBottomControlsBorderHeight = viewBottomControls.outerHeight (true) - viewBottomControls.height();
+    var viewContentBorderHeight = LayoutUtility.outerHeight (viewContentBorder) - viewContentBorder.clientHeight;
+    var viewBottomControlsBorderHeight = LayoutUtility.outerHeight (viewBottomControls) - viewBottomControls.clientHeight;
 
-    var viewContentOffset = viewContent.offset();
+    var viewContentOffset = LayoutUtility.offset (viewContent);
     var viewTop = viewContentOffset == null ? 0 : viewContentOffset.top;
 
-    var viewBottomControlsOffset = viewBottomControls.offset();
+    var viewBottomControlsOffset = LayoutUtility.offset (viewBottomControls);
     var bottomTop = viewBottomControlsOffset == null ? 0 : viewBottomControlsOffset.top;
 
     var viewNewHeight = bottomTop - viewTop - viewContentBorderHeight - viewBottomControlsBorderHeight;
 
-    viewContentBorder.height (viewNewHeight);
+    viewContentBorder.style.height = viewNewHeight + "px";
   };
 
-  public static AdjustSingleView (containerElement: JQuery): void
+  // TODO RM-7670 adjust signature
```

Comment:
This comment should have a short explanation of why we haven't adjusted yet/when it is going to be adjusted